src/calculator/UtilsClass.java

```diff
@@ -1,28 +1,67 @@
 package calculator;
 
+import com.mysql.cj.protocol.Resultset;
+
 import java.io.*;
+import java.sql.*;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.Properties;
 import java.util.Scanner;
 
-public class UtilsClass extends Calculator {
+public class UtilsClass {
     final static String RESULT_FILE = "D:\\IdeaProjects\\src\\calculator\\Results.txt";
+    final static String RESULT_COLUMN = "results";
+
+    public static void showPreviousResultsFromDB() {
+        Connection con = getDbConnection();
+        try {
+            Statement st = con.createStatement();
+            ResultSet result = st.executeQuery("SELECT * FROM `calculator_db`");
+            while (result.next()) {
+                double resFromDb = result.getDouble("results");
+                System.out.println(resFromDb);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    public static Connection getDbConnection() {
+        Connection conn = null;
+        Properties connectionProps = new Properties();
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
```

Comment:
This is deprecated and there is no need for it since java 7

Reply:
oh, i understand why i got the warnings now, thanks

Reply:
you mean Class.forName("com.mysql.jdbc.Driver"); right ?

Reply:
Yes